shared/wallets/transaction-details/index.js

```diff
@@ -99,6 +111,7 @@ const YourAccount = props => {
       counterpartyType={props.counterpartyType}
       counterparty={props.yourAccountName}
       accountID={yourAccountID}
+      onChat={() => {}}
```

Comment:
PR looks fine, except I think we'd usually try to avoid passing an empty function?  Like have `onChat?: string => void` instead and don't show the button if clicking it isn't going to do anything.

Reply:
Fixed this by splitting up the functions in a different way